### PR TITLE
[18.05] Release notes for 18.05.

### PR DESCRIPTION
**Pull request by jmchilton**
A good release! 😃

### CODE REVIEW

File: doc/source/releases/18.05.rst [hunk 1]
```diff
@@ -0,0 +1,604 @@
...
+  (thanks to `@nsoranzo <https://github.com/nsoranzo>`__).
+  `Pull Request 5786`_
+* Sniffing framework with constrained memory and I/O.
+  `Pull Request 5793`_
+* Infrastructure for dynamic compressed datatypes.
+  `Pull Request 5794`_
+* Capture more HDA state transitions in the history panel.
+  `Pull Request 5749`_
+* Add script for propagating hashtags in a Galaxy history
+  (thanks to `@bgruening <https://github.com/bgruening>`__).
```
Comment (nsoranzo) [inline doc/source/releases/18.05.rst]: This should be thanks to @anuprulez

File: doc/source/releases/18.05_announce.rst [hunk 2]
```diff
@@ -3,8 +3,78 @@
...
+* Support for chunked and resumable file uploads out-of-the-box. This should effectively
+  eliminate browser-based limitations on the size of files that can be uploaded to Galaxy.
+
+  `Pull Request 5516`_
+
+* The Galaxy UI features a powerful new widget for manipulating lists of rules for processing metadata
+  that can be used to describe the upload of arbitrarily complex collections as well as individual
+  datasets. These rules can be used to build collections from lists of URLs, FTP directory contents,
+  data library folders, and tabular sample sheets or exports from external data sources. The same
+  technique can be applied to the metadata of and existing collections enabling very flexible
```
Comment (nsoranzo) [inline doc/source/releases/18.05_announce.rst]: s/of and existing collections/of an existing collection/

**Comment by martenson** [comment]
wip list of what we need to add:

https://github.com/galaxyproject/galaxy/pull/5989
https://github.com/galaxyproject/galaxy/pull/6010
https://github.com/galaxyproject/galaxy/pull/5990
https://github.com/galaxyproject/galaxy/pull/6003
(after it is merged) https://github.com/galaxyproject/galaxy/pull/5997
after merge https://github.com/galaxyproject/galaxy/pull/6015



deprecate graphite #6018

if possible list new config options and datatypes as diff, example in: https://docs.galaxyproject.org/en/master/releases/17.05_announce.html#new-configuration-options

mention need for new charts repository

add warning about new compressed datatypes handling (custom datatypes_conf.xml could now contain missing compressed datatypes)

**Comment by martenson** [comment]
deprecate graphite https://github.com/galaxyproject/galaxy/pull/6018

**Comment by martenson** [comment]
Thank you both.

**Pull request merged**